History
=======

v0.1.1 / 2016-12-27
-------------------

* fix([#2](https://github.com/h2non/riprova/issues/2): handle and forward ``asyncio.CancelledError`` as non-retriable error.



v0.1.0 / 2016-12-25
-------------------

* First version
